mobify/platform-scaffold
# Im home and custom branch

This PR removes the home and custom branches from the scaffold Integration manager. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-21
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/688

## Changes
- Removed home and custom branches from the Integration Manager
- Updated paths for home and custom commands to point to the SDK versions

## How to test-drive this PR
- checkout the im-home-custom-branch branch for the SDK
- cd into the SDK folder
- run `npm link`
- run `npm run dev:build`
- cd back to the scaffold
- run `npm link progressive-web-sdk`
- run `npm run dev`
- preview the merlins site
- the home page should load
- add an item to the cart and proceed to the shipping page
- enter a known email address (jennifer@mobify.com) in the email field and tab out of it
- the login form should appear
